tests/specialops/tcallprecedence.nim

```diff
@@ -0,0 +1,80 @@
+discard """
+  nimout: '''
```

Comment:
since it's a new test, can you turn those into static/runtime `doAssert`'s instead of `nimout`? (makes it joinable, easier to read/edit, etc)